### DISCUSSION TITLE
5.1.1 版本中不能自动创建主题

### DISCUSSION THREAD

**Discussion by lyingfromyou** [apache/rocketmq#6830] [General]
5.1.1版本中, 我在broker.conf设置了autoCreateTopicEnable = true,  像重试或者死信主题依然不能自动创建, 像这种内置的规则主题名称, 由用户手动创建是否合理?
另, 官网文档中明确表示可以开启自动创建主题, 只是不推荐在生产环境中使用, 但是我在Issues中看到的回答是 该功能没有在proxy中实现

![image](https://github.com/apache/rocketmq/assets/24619864/8b41ba29-ce6f-439d-a74b-453e34d9e293)
![image](https://github.com/apache/rocketmq/assets/24619864/53b086a4-9576-478f-ac61-fb7951faf7a7)

**Comment by chenzlalvin** [comment]
The built-in system topic should not require manual creation by the user according to the design. We only recommend users to manually create topic for customer use. So please double-check your issue.

**Reply by lyingfromyou** [reply]
%RETRY%+consumerGroup 这种格式的主题是否应该由用户自己创建?
